index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Portfolio</title>
+	<link rel="stylesheet" type="text/css" href="Style.css">
+</head>
+<body>
+	<header>
+		<label class="logo">My Logo</label>
```

Comment:
The logo must be clickable. Please use an `<a>` tag for the logo. 
![image](https://user-images.githubusercontent.com/38088818/164516024-b1c49906-c2dd-49e6-b0e2-4e11c9889139.png)